nbextensions/usability/toc/main.js

```diff
@@ -6,9 +6,9 @@ define(["require", "jquery", "base/js/namespace"], function (require, $, IPython
   var make_link = function (h) {
     var a = $("<a/>");
     a.attr("href", '#' + h.attr('id'));
-    // get the text *excluding* the link text, whatever it may be
     var hclone = h.clone();
-    hclone.children().remove();
+    // last() is the |P anchor which lights up when you hoover over the headline
+    hclone.children().last().remove();
     a.text(hclone.text());
```

Comment:
Hi Jan, I think that there is perhaps a little better to do: try to replace the two .text() line 12 by .html(). This seems to enable to also render formatting in the link.